ceph_iscsi_config/lun.py

```diff
@@ -445,6 +460,11 @@ def allocate(self):
                                  "owner": self.owner,
                                  "pool": self.pool,
                                  "pool_id": rbd_image.pool_id}
+                    # add non-default control params to LUN config
+                    for k,v in self.controls.iteritems():
```

Comment:
I think this logic is only getting applied once at initial disk creation -- and not applied if the property is changed after the disk was added to the configuration. In the case where ```wwn``` is populated, that is where you would need to check for updates (and possibly delete the override if it's set to None).